tools/debugging/replay_wal.py

```diff
@@ -103,11 +136,22 @@ def replay_wal(storage, token_network_identifier, partner_address):
     '-p',
     '--partner-address',
 )
-def main(db_file, token_network_identifier, partner_address):
+@click.option(
+    '-x',
+    '--value-translator',
```

Comment:
Would it make sense to call this `address-translator` instead to give more clarify to the purpose?

Reply:
I changed the scope of this a little bit:

Addresses will now also match `pex()`ed addresses in the values (and `0x`-stripped ones).
It still allows to add other substitution rules (i.e. for non addresses). This can be useful to highlight things, e.g. `"identifier": "XXX"` makes it easier to spot identifiers in the output.

Apart from just translating things in the values, configured addresses will now also allow to reference the `-n` and `-p` flags (see description).

Therefore I changed the flag name to `--names-translator`.